### ISSUE TITLE
releases are confusing

### ISSUE THREAD

**Issue by 0xaf1f** [choishingwan/PRSice#78]
The latest release looks like it's 2.1.3.beta, which corresponds to https://github.com/choishingwan/PRSice/commit/ca28c81f23b6c7335ff4975bffb032648ca95f32. But in the [release notes for this version](https://github.com/choishingwan/PRSice/releases/tag/2.1.3.beta), you have an update log and a couple builds more recent than the actual release. Do these correspond to different commits? And if so, why aren't they separate patch releases?

Also, the [installation instructions](https://github.com/choishingwan/PRSice#installation) don't mention the releases and instead direct the installation from the tip of the `beta_testing` branch. Is this intentional or are these installation instructions meant for developers? I think if it's the latter, it should be labeled that way.

Many thanks for providing this software. I just hope the above can be clarified.

**Comment by 0xaf1f** [comment]
> Also, the installation instructions don't mention the releases

correction here -- they do, but it seems to refer to them just as an alternative to compiling from source. But the source one compiles from the git clone will surely be different than what was used to build the releases, right?

**Comment by choishingwan** [comment]
This is basically sloppy for me. Sometimes, I update the nightly build without rebasing the tag to the source code that I am using for the release. I am trying to think of a better way of doing this.

And the tip of the beta-testing will actually correspond to the current "developing" version, which can contain some bugs and stuff. I will definitely try to put the organisation of our release to the top of my to do list.

**Comment by choishingwan** [comment]
I have now updated the tag.

The thing is, the last digit represents the minor update and most of the time, we just have so much bug fix throughout that we might arrive something like 2.1.100 or something. And as these are all pre-releases, each time I change the version number, I'd also need to update the webpage. This causes some trouble (though maybe I can automate that). Hopefully, once we reached official release we can use the latest tag and by then we can use a better version numbering system (though I also hope we don't need to have that many bug fix)
